ivy/functional/backends/paddle/activations.py

```diff
@@ -58,6 +58,9 @@ def leaky_relu(
     return F.leaky_relu(x, negative_slope=alpha)
 
 
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("complex128", "complex64")}}, backend_version
+)
```

Comment:
How come we're marking this as unsupported?

Reply:
Which is essential for the approximation of gelu function
```python
def gelu(x): 
    # We don't have the exact implementation
    # cuz the erf function doesn't work on complex numbers
    sqrt_2_over_pi = ivy.sqrt(2 / ivy.pi).astype(x.dtype)
    x_pw = ivy.pow(x, 3)
    cdf = 0.5 * (1.0 + ivy.tanh(sqrt_2_over_pi * (x + 0.044715 * x_pw)))
    return x * cdf
    
```

Reply:
Ah, fair, so those limits would then be removed if/when `tanh` gets complex number support?